README.md

```diff
@@ -104,6 +104,12 @@ configuration. The following values are supported:
   - except the branch tip, which is exposed as
     `refs/heads/branches/$branch/tip`.
 
+These refs styles may also be combined in a comma-separated list. For example,
+a value of `bookmarks,heads,tips` is the same as `all`. A value of
+`bookmarks,tips` exposes the bookmarks and branch tips as
+`refs/heads/bookmarks/$bookmark` and `refs/heads/branches/$branch`,
+respectively.
```

Comment:
I would rearrange this some more. Starting from
> The following values are supported
becoming something like
> The following values may be combined

Then removing the `all` item, and add a paragraph explaining how the refs vary depending on the combinations, and that the combination of all of them is the default.

Reply:
Ok, I rewrote this a bit to be clear about all combinations, hope that's better